app/repositories/ITSAStatusRepositorySupport.scala

```diff
@@ -23,12 +23,13 @@ object ITSAStatusRepositorySupport {
 
   def statusToString(status: ITSAStatus): String =
     status match {
-      case ITSAStatus.NoStatus  => "U"
+      //Making all the valid status as Unknown until we get clarity on what to do with these status
+      case ITSAStatus.NoStatus | ITSAStatus.Exempt | ITSAStatus.DigitallyExempt | ITSAStatus.Dormant  => "U"
```

Comment:
Looks like this is the area where the test is failing from.

it's no longer calling a runtime exception due to those valid statuses being set to unknown